control/cephutils.py

```diff
@@ -64,6 +73,14 @@ def get_number_created_gateways(self, pool, group):
                 rply = self.execute_ceph_monitor_command(str)
                 self.logger.debug(f"reply \"{rply}\"")
                 conv_str = rply[1].decode()
+                data = json.loads(conv_str)
```

Comment:
No point in doing json.loads() unless find is not -1, you can move the line into the if.